src/vcpkg/configure-environment.cpp

```diff
@@ -0,0 +1,81 @@
+#include <vcpkg/base/basic_checks.h>
+#include <vcpkg/base/downloads.h>
+#include <vcpkg/base/system.print.h>
+#include <vcpkg/base/system.process.h>
+
+#include <vcpkg/archives.h>
+#include <vcpkg/commands.version.h>
+#include <vcpkg/configure-environment.h>
+#include <vcpkg/tools.h>
+#include <vcpkg/vcpkgcmdarguments.h>
+#include <vcpkg/vcpkgpaths.h>
+
+namespace vcpkg
+{
+    int run_configure_environment_command(const VcpkgPaths& paths, View<std::string> args)
+    {
+        print2(Color::warning, "vcpkg-ce ('configure environment') is experimental and may change at any time.\n");
```

Comment:
We might want to extract this out to a function (see above comment)

Reply:
I'm still not sure what that accomplishes. All the CE things need to print the message and they do this way without callers needing to remember to to call something else. When do we want that message without actually invoking CE?

(I'm all for extracting things into smaller functions; this is more a reply about the other bit saying that all callers of this thing should call something else first that prints the message first)